src/components/QuickActions.vue

```diff
@@ -1,18 +1,66 @@
 <script setup lang="ts">
-import { quickActions } from '../utilities/helper'
+import heartIcon from '../assets/icons/heart-icon.svg'
+import savedIcon from '../assets/icons/saved-icon.svg'
+import corusesIcon from '../assets/icons/courses-icon.svg'
+import settingsIcon from '../assets/icons/setting-icon.svg'
+import logoutIcon from '../assets/icons/logout-icon.svg'
 import { useAuthStore } from '../stores/auth.ts'
 import { isMobile } from '../utilities/helper'
 import { useRouter } from 'vue-router'
 
 const router = useRouter()
-const quickActionBtns = quickActions()
 const authStore = useAuthStore()
 
-const logout = (quickAction: any) => {
-  if (quickAction.name === 'Logout') {
-    authStore.logout(router)
+const logout = () => {
+ if (window.confirm("Are you sure you want to log out?")) {
+    authStore.logout(router);
   }
 }
+
+function quickActions() {
```

Comment:
Why is there a need to create a method which returns quickAction buttons array? 
Can we not directly assign the array to the quickActionBtns variable?